meshroom/core/node.py

```diff
@@ -940,10 +940,15 @@ def endSequence(self):
         pass
 
     def stopComputation(self):
-        """ Stop the computation of this node. """
-        for chunk in self._chunks.values():
-            if not chunk.isExtern():
+        """ Stop the computation of this node. Chunks that are running locally will be stopped, and those that
+        are submitted locally will be reset.
+        """
+        for chunk in [c for c in self._chunks.values() if c.status.status == Status.RUNNING or
+                                                          c.status.status == Status.SUBMITTED]:
+            if not chunk.isExtern() and chunk.status.status == Status.RUNNING:
                 chunk.stopProcess()
+            elif not chunk.isExtern():
+                chunk.upgradeStatusTo(Status.NONE)
```

Comment:
```suggestion
            if not chunk.isExtern():
                if chunk.status.status == Status.RUNNING:
                    chunk.stopProcess()
                else:
                    chunk.upgradeStatusTo(Status.NONE)
```

Reply:
Done: the double `for` loop has been removed, and the `else` from this suggestion has been replaced by an `elif chunk.status.status == Status.SUBMITTED` as it otherwise all the chunks would be reset, including those which are in the `SUCCESS` state.